plugins/genesys/fu-genesys-usbhub-device.c

```diff
@@ -2161,13 +2235,389 @@ fu_genesys_usbhub_device_update_firmware(FuGenesysUsbhubDevice *self,
 	fu_progress_step_done(progress);
 
 	/* verify */
+	if (!self->has_hw_codesign) { /* skip specific codesign verification */
+		if (!fu_genesys_usbhub_device_compare_flash_data(self,
+								 start_addr,
+								 buf,
+								 bufsz,
+								 fu_progress_get_child(progress),
+								 error))
+			return FALSE;
+	}
+	fu_progress_step_done(progress);
```

Comment:
don't you need a fu_progress_add_step() for this case? Otherwise you'll get a warning about going over 100%.

Reply:
Existed at https://github.com/fwupd/fwupd/blob/0fb8267c7ad9b0be834ce181d33f66bfc3ed82c9/plugins/genesys/fu-genesys-usbhub-device.c#L2206